ambry-router/src/test/java/com.github.ambry.router/PutManagerTest.java

```diff
@@ -438,46 +422,26 @@ public void testBadChannelPutFailure() throws Exception {
   }
 
   /**
-   * A bad arguments test, where the channel size is different from the size in BlobProperties.
-   */
-  @Test
-  public void testChannelSizeNotSizeInPropertiesPutFailure() throws Exception {
-    requestAndResultsList.clear();
-    int blobSize = chunkSize;
-    RequestAndResult requestAndResult = new RequestAndResult(blobSize);
-    // Change the actual content size.
-    requestAndResult.putContent = new byte[blobSize + 1];
-    requestAndResultsList.add(requestAndResult);
-    Exception expectedException = new RouterException("", RouterErrorCode.BadInputChannel);
-    submitPutsAndAssertFailure(expectedException, true, false);
-  }
-
-  /**
-   * A bad arguments test, where the blob size is very large.
+   * Test that the size in BlobProperties is ignored for puts, by attempting puts with varying values for size in
+   * BlobProperties.
    */
   @Test
-  public void testBlobTooLargePutFailure() throws Exception {
```

Comment:
Yeah - it is hard, as all the chunks have to be filled (and the mock servers have to keep them) and it will take a while. I did toy with the idea of having an explicit max size config in the router that will determine when a blob is too large. I decided against it at this time.